sdk/python/tests/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 kubeflow.org
```

Comment:
are these files needed if we don't have any code in them?

Reply:
yes, Python uses the mere existing of the `__init__.py` files to indicate a folder being a Python package